server/src/main/java/org/opentosca/toscana/plugins/cloudfoundry/visitors/NodeVisitors.java

```diff
@@ -76,6 +86,27 @@ public void visit(Apache node) {
     @Override
     public void visit(WebApplication node) {
         myApp.setName(node.getEntityName());
+
+        StandardLifecycle lifecycle = node.getStandardLifecycle();
+        Optional<Operation> configureOptional = lifecycle.getConfigure();
+        
+        //get configure script
+        if (configureOptional.isPresent()) {
+            Optional<Artifact> configureArtifact = configureOptional.get().getArtifact();
+            if (configureArtifact.isPresent()) {
```

Comment:
According to my IntelliJ IDEA:
>Can be replaced with single expression in functional style.
`configureArtifact.ifPresent(artifact -> myApp.addExecuteFile(artifact.getFilePath(), node));`